packages/modelviewer.dev/data/docs.json

```diff
@@ -395,77 +367,69 @@
         "htmlName": "cameraOrbit",
         "description": "Set the starting and/or subsequent orbital position of the camera. You can control the azimuthal and polar angles, and the distance from the model. Accepts values of the form \"$angle $angle $distance\", like \"10deg 75deg 1.5m\". Also supports units in radians (\"rad\") for angles and centimeters (\"cm\") or millimeters (\"mm\") for camera distance. Camera distance can also be set as a percentage ('%'), where 100% gives the model tight framing within any window based on all possible theta and phi values. Any time this value changes from its initially configured value, the camera will interpolate from its current position to the new value. Any value set to 'auto' will revert to the default. For camera-orbit, camera-target and field-of-view, parts of the property value can be configured with CSS-like functions. The CSS calc() function is supported for these values, as well as a specialized form of the env() function. You can use env(window-scroll-y) anywhere in the expression to get a number from 0-1 that corresponds to the current top-level scroll position of the current frame. For example, a value like \"calc(30deg - env(window-scroll-y) * 60deg) 75deg 1.5m\" cause the camera to orbit horizontally around the model as the user scrolls down the page.",
         "default": {
-          "type": "$angle $angle $distance",
           "default": "0deg 75deg 105%",
-          "options": "Multiple forms described in description"
+          "options": "multiple forms described in description"
         }
       },
       {
         "name": "camera-target",
         "htmlName": "cameraTarget",
         "description": "Set the starting and/or subsequent point the camera orbits around. Accepts values of the form \"$X $Y $Z\", like \"0m 1.5m -0.5m\". Also supports units in centimeters (\"cm\") or millimeters (\"mm\"). A special value \"auto\" can be used, which sets the target to the center of the model's bounding box in that dimension. Any time this value changes from its initially configured value, the camera will interpolate from its current position to the new value. Defaults to \"auto auto auto\".",
         "default": {
-          "type": "$X $Y $Z",
           "default": "auto auto auto",
-          "options": "Multiple forms described in description"
```

Comment:
Instead of "multiple forms", for ones like this I think what we had in the type was more descriptive and we can just move it to options.

Reply:
done

Reply:
good call--